CONTRIBUTING.md

```diff
@@ -40,6 +40,21 @@ Before submitting a pull request, [rebase](https://www.atlassian.com/git/tutoria
 
 Learn more about [submitting pull requests](https://microsoft.github.io/fast-dna/docs/en/contributing/working).
 
+### Documenting breaking changes
+
+Make sure to document the migration strategy in a `MIGRATION.md` file in the package(s) that has breaking changes.
```

Comment:
Assuming we want this in the _root_. Do we want to specify location? Is it helpful to be explicit?

Reply:
A breaking change only affects the package that the breaking change is in, so it should be localized to packages.

Reply:
Or did you mean I should provide an example of where it would go?

Reply:
Yeah, more of that. Is it helpful to show a path like:
```
packages/fast-components-styles-msft/MIGRATION.md
```

Reply:
not explcitly saying it *is* - more of a question

Reply:
Added